packages/contract/src/test-utils/test-node-launcher.test.ts

```diff
@@ -0,0 +1,254 @@
+import { Provider, sleepUntilTrue } from '@fuel-ts/account';
+import { WalletConfig } from '@fuel-ts/account/test-utils';
+import * as launchCustomProviderAndGetWalletsMod from '@fuel-ts/account/test-utils';
+import { FuelError } from '@fuel-ts/errors';
+import { expectToThrowFuelError, safeExec } from '@fuel-ts/errors/test-utils';
+import type { ChainConfig } from '@fuel-ts/utils';
+import { randomInt, randomUUID } from 'crypto';
```

Comment:
As above, please use internal crypto package.

Reply:
`randomUUID` and `randomInt` aren't part of the `@fuel-ts/crypto` package currently. I'd rather have that done in another PR and leave this as it is right now.

Reply:
Should we have a `TODO` tag?

Reply:
Created issue https://github.com/FuelLabs/fuels-ts/issues/1807 for `randomUUID`, while `randomInt` doesn't have a browser crypto equivalent. I'd leave `randomInt` here as it is because this is only a test file that's for node-based tests anyways.